app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -225,9 +222,12 @@ export const ContentForm: React.FC<IContentFormProps> = ({
         channel('content', result);
 
         if (!originalId) {
-          // Reset form for next record.
-          setForm({ ...defaultFormValues(contentType) });
-          // navigate(getContentPath(combined, contentResult.id, contentResult?.contentType));
+          if (result.status === ContentStatusName.Draft)
```

Comment:
Only start a new item if the original was published.  This enables saving content and continuing to edit it.  Important for uploading files currently.